deploy/csi-daemonset.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: mayastor
+  name: mayastor-csi
+  labels:
+    openebs/engine: mayastor
```

Comment:
(if) I read this correctly, deploying the csi plugin to a node will cause this label to be applied to that node?  If so, does this then not cause mayastor to be deployed there too (as this label is a selector clause for the mayastor daemonset)?

Reply:
it does not apply the label to the node but rather to the daemonset object. So we should be fine here 👍

Reply:
Withdrawing my previous comment.  It is the Daemonset object definition which is being labeled, not the resultant pods, so there is no issue.